Dockerfile

```diff
@@ -76,12 +76,18 @@ WORKDIR /srv/sylius
 ARG SYMFONY_ENV=prod
 
 # prevent the reinstallation of vendors at every changes in the source code
-COPY composer.json composer.lock ./
+COPY composer.json composer.lock symfony.lock ./
 RUN set -eux; \
 	composer install --prefer-dist --no-autoloader --no-scripts --no-progress --no-suggest; \
 	composer clear-cache
 
-COPY . ./
+# copy only specifically what we need
```

Comment:
We do not copy directories like `features` and `tests`, because they're not used in production. But they're available in development anyway, since we bind-mount the project directory from host.